backend/lib/deal-observer.js

```diff
@@ -18,7 +18,7 @@ export const observeBuiltinActorEvents = async (pgPool, makeRpcRequest, maxPastE
   const lastInsertedDeal = await fetchDealWithHighestActivatedEpoch(pgPool)
   const startEpoch = Math.max(
     currentChainHead.Height - maxPastEpochs,
-    (lastInsertedDeal?.activated_at_epoch + 1) || 0
+    ((lastInsertedDeal?.activated_at_epoch ?? -1) + 1) || 0
```

Comment:
```suggestion
    (lastInsertedDeal?.activated_at_epoch ?? -1) + 1
```

The `|| 0` isn't necessary any more now